### ISSUE TITLE
rev2 Modeling and Training

### ISSUE THREAD

**Issue by obadx** [obadx/muaalem-paper#7]
* [x] Provide a compact mathematical description: encoder $E_{\theta}(x)$, 11 linear heads $W_{k}$, CTC losses $L_{k}$, and the final weighted loss $L = \sum w_{k}L_{k}$.
* [x] Elaborate how the weights of each level is assigned as its confusing in the current setup
* [x] Report tarin/dev loss curves to show that the model did not overfit
* [x] Mention the exact parameters dropout and adapter layers but I should mention that
* [x] Mention ruining an experiment with SpechAugment but was very bad
